lib/stat_tracker.rb

```diff
@@ -25,8 +25,35 @@ def self.from_csv(locations)
   #   end
   # end
 
+
+  def low_ave_score_away
+    @games_data.group_by {|row| row["away_team_id"]}.map do |tid, scores|
+      {tid => scores.sum {|score| score["away_goals"].to_i}.to_f/ scores.length} 
+    end 
+  end
+
+  def low_ave_score_team
+    low_ave_score_away.min_by{|hash| hash.values}.keys
+  end
+
+  def team_id_to_name
+    @teams_data.map do |row|
+      {row["team_id"] => row["teamName"]}
+    end
+  end
+
+  def lowest_scoring_visitor
+    team_id_to_name.find {|pairs| pairs.find {|key, value| p value if key == low_ave_score_team[0]}}.values[0] 
```

Comment:
I think we can remove "p value if" on this line.  I noticed that when the spec_harness was running it was outputting the name of the team (instead of just the green dot for passing) and i think that's because we're putsing here. The find enumerator will return the value for us automatically once it meets the condition in the block.

Reply:
I deleted 'p value if' and pushed my changes to a new PR along with adding the lowest scoring home team method. These changes are on the ep/feat/lowest_scoring_hometeam branch. Also pushed these changes on this branch so we can merge this branch into main with working methods.